Description: skip tests when needed feature is disabled
Author: Jonas Smedegaard <dr@jones.dk>
Last-Update: 2023-01-26
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/config.rs
+++ b/src/config.rs
@@ -1309,6 +1309,7 @@
 
 #[test]
 #[serial]
+#[cfg(feature = "s3")]
 fn test_s3_no_credentials_conflict() {
     env::set_var("SCCACHE_S3_NO_CREDENTIALS", "true");
     env::set_var("SCCACHE_BUCKET", "my-bucket");
@@ -1390,6 +1391,7 @@
 }
 
 #[test]
+#[cfg(feature = "gcs")]
 fn test_gcs_service_account() {
     env::set_var("SCCACHE_GCS_BUCKET", "my-bucket");
     env::set_var("SCCACHE_GCS_SERVICE_ACCOUNT", "my@example.com");
